docs/cristin-proxy-swagger.yaml

```diff
@@ -2417,6 +2417,9 @@ components:
           description: Employments for this person. Visible to authorized users
           items:
             $ref: '#/components/schemas/PersonEmployment'
+        verified:
+          type: boolean
+          description: If the person is a verified person with a national identity number
```

Comment:
I'm not sure that this will hold over time — perhaps write "Whether we can document the existence of the person"

Reply:
Foreign workers might get new national ID, depending on what kind of ID they are entering our border with,